### ISSUE TITLE
[teddit error / not related to SearXNG] Theme Style 'auto' setting triggers error

### ISSUE THREAD

**Issue by cameronj86** [searxng/searxng#3090]
**Version of SearXNG**
Docker Compose 2023.9.25+1f835a9f1
**What happened?**
Theme style breaks when set on _auto_. When I check the container logs it is attempting to use the color scheme 'nord' for auto's dark mode but cannot find it. Couldn't find any clear references to nord in Github docs to try and manually adjust it. 
**How To Reproduce**
Use auto-mode (perhaps @ night but I think the error re: settings/config pops up all the time)
**Expected behavior**
SearXng page switches color based on system theme and/or time of day
**Additional context**
If I manually change between light/dark mode, there are no issues. This makes me think that dark mode's theme != auto's dark mode theme, and might just a theme name update.

This seems like it would be a bug that most docker users come across since the default setting for theme style is _auto_ IIRC, so it seems weird that there's no other tix loaded. This makes me slightly question whether SearXng is trying to match a theme for my system, but that seems low probability.

**Comment by return42** [comment]
> Theme style breaks when set on auto. When I check the container logs it is attempting to use the color scheme 'nord' for auto's dark mode but cannot find it. 

Then you have not installed some customizing that uses the [nord-theme](https://www.nordtheme.com/).  Please contact the author of your customizing.

**Comment by cameronj86** [comment]
Following up since the ticket was disregarded and the explanation was incorrect. I was (and still am) using default settings.

The issue is likely that redis is using the same port on another running docker container and the two are sending conflicting info re: theming.

**Comment by return42** [comment]
> Following up since the ticket was disregarded and the explanation was incorrect. I was (and still am) using default settings.

Not I fully understand you, but you wrote

> it is attempting to use the color scheme 'nord' for auto's dark mode but cannot find it.

.. and 'nord' is nowhere in SearXNG .. its not a part of, so I asking myself from where does this com .. may you better post the log you referred. 

> The issue is likely that redis is using the same port on another running docker container and the two are sending conflicting info re: theming.

Now I'm completely confused .. the title of this post is "Theme Style 'auto' setting triggers error" .. and with the last comment you say you have conflicting docker containers?!?

**Comment by cameronj86** [comment]
![Screenshot_20240220_133543](https://github.com/searxng/searxng/assets/8635023/851bed02-923b-48bb-a501-5b0e727a1ef5)

```
Traceback (most recent call last):
  File "/usr/local/searxng/searx/webapp.py", line 488, in pre_request
    preferences.parse_dict(request.cookies)
  File "/usr/local/searxng/searx/preferences.py", line 516, in parse_dict
    self.key_value_settings[user_setting_name].parse(user_setting)
  File "/usr/local/searxng/searx/preferences.py", line 93, in parse
    self._validate_selection(data)
  File "/usr/local/searxng/searx/preferences.py", line 89, in _validate_selection
    raise ValidationException('Invalid value: "{0}"'.format(selection))
searx.preferences.ValidationException: Invalid value: "auto"
2024-02-20 19:32:16,802 WARNING:searx.engines.duckduckgo: ErrorContext('searx/search/processors/online.py', 118, "response = req(params['url'], **request_args)", 'searx.exceptions.SearxEngineAccessDeniedException', None, ('HTTP error 403',)) False
2024-02-20 19:32:16,802 ERROR:searx.engines.duckduckgo: Searx is blocked
Traceback (most recent call last):
  File "/usr/local/searxng/searx/search/processors/online.py", line 165, in search
    search_results = self._search_basic(query, params)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/searxng/searx/search/processors/online.py", line 149, in _search_basic
    response = self._send_http_request(params)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/searxng/searx/search/processors/online.py", line 118, in _send_http_request
    response = req(params['url'], **request_args)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/searxng/searx/network/__init__.py", line 179, in post
    return request('post', url, data=data, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/searxng/searx/network/__init__.py", line 96, in request
    return future.result(timeout)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/concurrent/futures/_base.py", line 456, in result
    return self.__get_result()
           ^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/concurrent/futures/_base.py", line 401, in __get_result
    raise self._exception
  File "/usr/local/searxng/searx/network/network.py", line 291, in request
    return await self.call_client(False, method, url, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/searxng/searx/network/network.py", line 274, in call_client
    return Network.patch_response(response, do_raise_for_httperror)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/searxng/searx/network/network.py", line 247, in patch_response
    raise_for_httperror(response)
  File "/usr/local/searxng/searx/network/raise_for_httperror.py", line 75, in raise_for_httperror
    raise SearxEngineAccessDeniedException(message='HTTP error ' + str(resp.status_code))

```

Error goes away when I turn the theme from `auto` to `light`:
![image](https://github.com/searxng/searxng/assets/8635023/392662f1-60c4-4c36-b215-7e4ade673a65)

Previously, the error message was related to `Nord`, and the only reference point that I have is that `nord` runs as a theme on my personal teddit instance:
![image](https://github.com/searxng/searxng/assets/8635023/de89d73d-3255-413c-9c38-c596d659ba2f)

The docker containers are completely separate but are using the default redis port (6379) and so my uneducated guess is that the they both have competing  theme variables with the same naming convention. I did a search and couldn't find any examples or solutions of this in the past though. 

**So to summarize:** 
SearXNG has a theme that can be set to `simple`. That is the only option available.
Teddit has a theme that can be set to various options, two of which are causing my SearXNG to fail (`nord` & `auto`)

How/why is SearXNG going across docker containers to get these variables? The only commonality that I can pinpoint is the redis port.

**Comment by cameronj86** [comment]
Final confirmation:

Updated Settings to `sepia` on teddit
![image](https://github.com/searxng/searxng/assets/8635023/1fb7cfbc-443c-455a-9f59-aeae8bb7c2ae)

New error log on SearXNG after search:
```
searx.preferences.ValidationException: Invalid value: "sepia"
2024-02-20 19:50:47,014 ERROR:searx.webapp: Invalid value: "sepia"
Traceback (most recent call last):
  File "/usr/local/searxng/searx/webapp.py", line 488, in pre_request
    preferences.parse_dict(request.cookies)
  File "/usr/local/searxng/searx/preferences.py", line 516, in parse_dict
    self.key_value_settings[user_setting_name].parse(user_setting)
  File "/usr/local/searxng/searx/preferences.py", line 93, in parse
    self._validate_selection(data)
  File "/usr/local/searxng/searx/preferences.py", line 89, in _validate_selection
    raise ValidationException('Invalid value: "{0}"'.format(selection))
searx.preferences.ValidationException: Invalid value: "sepia"
```

**Comment by return42** [comment]
> Teddit has a theme that can be set to various options, two of which are causing my SearXNG to fail (`nord` & `auto`)

Don't use teddit if it breaks other applications / BTW: [teddit is no more actively maintained!](https://codeberg.org/teddit/teddit#teddit-is-no-more-actively-maintained)
